src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -139,6 +139,10 @@ public function getConfigTreeBuilder()
                                 }
 
                                 if (false !== strpos($v, '/')) {
+                                    if ('0.0.0.0/0' === $v) {
+                                        return false;
+                                    }
```

Comment:
@xabbuh from another hand, the `'0.0.0.0/0'` scenario is less likely to happen. So how about promoting the more-likely-to-happen scenario over the other?

If we move this check before the `if (false !== strpos($v, '/')) {` then we would have 2 checks for almost every symfony user, instead of just 1 now.